src/gui/sdlmain.cpp

```diff
@@ -1308,6 +1309,7 @@ void GFX_ToggleMouseCapture(void) {
 	        "SDL: Mouse capture is invalid when NoMouse is configured [Logic Bug]");
 
 	mouse_is_captured = mouse_is_captured ? SDL_FALSE : SDL_TRUE;
+	SDL_SetHint(SDL_HINT_MOUSE_RELATIVE_MODE_WARP, sdl.mouse.raw_input ? "0" : "1");
```

Comment:
This hint is placed inside the MouseToggleCapture function, which fires many times throughout DOSBox's lifetime,  each time when we need to either capture or release the mouse. 

However, this `SDL_SetHint(SDL_HINT_MOUSE_RELATIVE_MODE_WARP` now re-applies the same setting over-and-over inside this MouseToggle function.  We don't need this repeating over-and-over here; and it's also not related to capturing or releasing the mouse. 

Therefore, I would suggest relocating `SDL_SetHint(SDL_HINT_MOUSE_RELATIVE_MODE_WARP` to a single call on startup. (see below)

Reply:
One more request about this line: if `SDL_SetHint` failed, log a message, maybe `SDL: Setting mouse raw input failed`.

Reply:
Actually, you do need to re-apply this every time since the parameter can be changed at runtime. We can make it unchangeable, though.

Reply:
Suggest it be made unchangeable, and move the hint higher up so it's called once for the lifetime of the dosbox instance.